File: c/1065.c
#include <stdio.h>

int main(){

    int e, i, cont=0;
    for (i=0;i<5;i++){
        scanf("%d", &e);
        if (e%2==0){
            cont++;
        }
    }
    printf("%d valores pares\n", cont);
}

File: js/PROBLEMS/1065.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(function (item){
    return parseInt(item)
})

cont = 0;
for(line in lines){
    if (lines[line]%2 == 0){
        cont++;
    }
}

console.log(cont+" valores pares")